install_config/install/advanced_install.adoc

```diff
@@ -1865,13 +1865,22 @@ availability (HA) method:
 combined with any load balancing solution. If a host is defined in the *[lb]*
 section of the inventory file, Ansible installs and configures HAProxy
 automatically as the load balancing solution. If no host is defined, it is
-assumed you have pre-configured a load balancing solution of your choice to
+assumed you have pre-configured an external load balancing solution of your choice to
 balance the master API (port 8443) on all master hosts.
 |===
 
-For your pre-configured load balancing solution, you must have:
+[NOTE]
+====
+This HAProxy load balancer is intended to demonstrate the API server's HA mode 
+and is not recommended for production environments. If you are deploying to a cloud provider we recommend 
+that you deploy a cloud-native TCP-based load balancer or take other steps to provide a highly available load balancer.
```

Comment:
"We" is second-person, and not recommended in the docs. Maybe:

"If you are deploying to a cloud provider, Red Hat recommends deploying a cloud-native TCP-based load balancer, or taking other steps to provide a highly available load balancer."